src/heap/heap.cc

```diff
@@ -1744,6 +1744,91 @@ Heap::DevToolsTraceEventScope::~DevToolsTraceEventScope() {
                    heap_->SizeOfObjects());
 }
 
+inline std::string get_env(const std::string& env_name) {
+  char* ret = getenv(env_name.c_str());
+  return ret ? std::string(ret) : std::string();
+}
+
+inline bool use_membalancer() {
+  return get_env("USE_MEMBALANCER") == "1";
+}
```

Comment:
Let's add a comment above this saying that this is a stub and should ideally integrate with Chrome's flags/configuration/parameters system.